README.md

```diff
@@ -0,0 +1,53 @@
+## orbit-db-cache
+
+Isomorphic cache used by `orbit-db`, implemented using `level.js` (indexedDB) for the browser and `leveldown` (leveldb) for node.
+
+### Used in
+
+* [orbit-db](https://github/com/orbitdb/orbit-db)
+
+### API
+
+#### Public methods
+
+##### `open`
+
+Open cache store and make it available to be used (indexedDB or leveldb)
+
+##### `close`
+
+Close cache store and delete cached items
```

Comment:
Calling `close()` will close the cache but it doesn't delete cached items locally, ie. the leveldb database is not deleted upon close. I think we can write "Close the cache and the underlying leveldb instance".

Reply:
Got it, `Close the cache and the underlying indexedDB / leveldb instance`. Is that alright?